docs/plugins/google-cloud.md

```diff
@@ -88,6 +88,7 @@ googleCloud({
       '@opentelemetry/instrumentation-net': { enabled: false },
     },
     metricExportIntervalMillis: 5_000,
+    metricExportTimeoutMillis: 5_000,
```

Comment:
The default values for both of these fields is 300_000. I'd suggest having this example use those defaults explicitly. Using 5s intervals is useful during development, but could end up using a lot more metric quota than is needed if deployed to production.

My guess is that this example was failing because the export interval would be lower than the default timeout.

Reply:
@chees if you can make the change above, we'll get this merged in asap. Thanks!